collector/installation.go

```diff
@@ -66,8 +54,8 @@ func (i Installation) Collect(c *CollectorOpts) interface{} {
 		log.Errorf("Failed to get authProviders err=%s", err)
 	}
 
-	log.Debug("Collecting Users")
-	userList, err := c.Client.User.List(&nonRemoved)
+	log.Debug("  Collecting Users")
```

Comment:
Why a space in all these ?

Reply:
To format debug messages. I tried to "normalize" it like it was on other collectors

Reply:
Raul showed me, its like a nested format now in logs.

Reply:
Yeah, it's just for debug messages to have an easy visual understanding, like..

```
INFO[2020-02-12 20:15:14] Collecting anonymous data from https://XXXXXX/v3
DEBU[2020-02-12 20:15:16] Collecting Clusters
DEBU[2020-02-12 20:15:16]   Found 3 Clusters
DEBU[2020-02-12 20:15:16]   Cluster: ec2-cluster
DEBU[2020-02-12 20:15:16]     CPU cores=10, util=73
DEBU[2020-02-12 20:15:16]     Mem used=7201, total=39411, util=18
DEBU[2020-02-12 20:15:16]     Pod used=75, total=550, util=14
DEBU[2020-02-12 20:15:18]   Cluster: 2-windows
DEBU[2020-02-12 20:15:18]     CPU cores=7, util=6
DEBU[2020-02-12 20:15:18]     Mem used=270, total=29178, util=1
DEBU[2020-02-12 20:15:18]     Pod used=15, total=330, util=5
DEBU[2020-02-12 20:15:19]   Cluster: local
DEBU[2020-02-12 20:15:19]     CPU cores=24, util=11
DEBU[2020-02-12 20:15:19]     Mem used=1470, total=95734, util=2
DEBU[2020-02-12 20:15:19]     Pod used=26, total=330, util=8
DEBU[2020-02-12 20:15:21] Collecting Installation
DEBU[2020-02-12 20:15:22]   Installation Server Version: v2.3.4
DEBU[2020-02-12 20:15:22]   Collecting AuthConfigs
DEBU[2020-02-12 20:15:22]   Collecting Users
DEBU[2020-02-12 20:15:22]   Collecting NodeDrivers
DEBU[2020-02-12 20:15:22]   Collecting KontainerDrivers
DEBU[2020-02-12 20:15:23]   Looking for Local cluser
DEBU[2020-02-12 20:15:23] Collecting MultiClusterApps
DEBU[2020-02-12 20:15:23]   Found 0 MultiClusterApps
DEBU[2020-02-12 20:15:23]   Collecting DNS Providers
DEBU[2020-02-12 20:15:23]     Found 0 DNS Providers
DEBU[2020-02-12 20:15:23]   Collecting DNS Entries
DEBU[2020-02-12 20:15:23]     Found 0 DNS Entries
DEBU[2020-02-12 20:15:23] Collecting Nodes
DEBU[2020-02-12 20:15:24]   Found 37 Nodes
DEBU[2020-02-12 20:15:24]   Node: fe-demo-ec2-cpetcd-3
DEBU[2020-02-12 20:15:24]     CPU cores=2, util=18
DEBU[2020-02-12 20:15:24]     Mem used=30, total=7882, util=0
DEBU[2020-02-12 20:15:24]     Pod used=5, total=110, util=5
```